src/components/dictionaryConcepts/containers/DictionaryConcepts.jsx

```diff
@@ -50,7 +50,7 @@ export class DictionaryConcepts extends Component {
     this.state = {
       conceptsCount: this.props.totalConceptCount,
       searchInput: '',
-      conceptLimit: 10,
+      conceptLimit: 20,
```

Comment:
Can this be done in a different ticket? `Increase the concept limit to 20 with a reason`. Unless it is part of this ticket's description.

Reply:
@dondrzzy  I branched off from this branch OCLOMRS-252: Add all bulkconcepts fix. This is a mistake. Because this PR has only two changes.

Reply:
@Michyjones, why did you branch off another person's branch?

Reply:
@dondrzzy thanks for that. I always branch from the master branch. This was a mistake I believe it will never happen again.